package.json

```diff
@@ -17,7 +17,8 @@
     "ci:lint": "eslint 'src/**/*.js' -c ./.eslintrc.js",
     "ci:test": "jest --ci",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "prettier": "prettier 'src/**/*.js' --write"
```

Comment:
```suggestion
    "prettier": "prettier --write ."
```

Reply:
In this case we need to exclude a lot of things that are in the root

Reply:
I think we can use something like:

```
"prettier": "prettier --write '{src,tests}/**/*.{js,md}' './*.{js,md}'"
```

This will format all `js` and `md` files in the `./src` and `./tests` folders. And all `js` and `md` files in the root.

Reply:
Why do we need to exclude files? 🤔

Prettier will know whether or not it can prettify a certain file or not

Reply:
Yes we can, but how many folders we need to exclude ?

Reply:
None, Prettier will run on all files and will see "Oh I can't prettify this one" and will just go to the next file

Reply:
`node_modules`, `.cache`, `dist`, I think those are the ones that need to be ignored?

Reply:
Yes I think so

Reply:
A `.prettierignore` file will indeed handle those directories

Reply:
Sounds like a plan :grin: just lint it all. :+1: Create a `.prettierignore` based on the current `.gitignore`, and add directories if you see anything weird happening.

The positive side of blacklisting instead of whitelisting is that things like cypress (#27) are than also automatically included.

I only wished that `.xignore` files could import each other :grin:

Reply:
> I only wished that .xignore files could import each other

I did find this one, looks interesting :exploding_head: 

https://www.npmjs.com/package/ignore-sync